__tests__/app.test.js

```diff
@@ -94,18 +94,20 @@ describe('GET /api/articles/:article_id', () => {
 			.get('/api/articles/1')
 			.expect(200)
 			.then(({ body }) => {
-				const { article } = body;
-				expect(article).toEqual({
-					article_id: 1,
-					title: 'Living in the shadow of a great man',
-					topic: 'mitch',
-					author: 'butter_bridge',
-					body: 'I find this existence challenging',
-					created_at: '2020-07-09T20:11:00.000Z',
-					votes: 100,
-					article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
-				});
-			});
+                const { article } = body;
```

Comment:
This would be better as a new test, but I can see how this would break your old test - in future it may be worth using a more flexible jest matcher such as `toMatchObject` to avoid this.